.yourbase.yml

```diff
@@ -19,8 +19,9 @@ build_targets:
       - apt-get install -yq rsync
       - git config --global user.name "Garden CI"
       - git config --global user.email "admin@garden.io"
-      - cd garden-service
       - npm install
```

Comment:
I thought this was already done by line 8.. but then I discovered that npm ci actually nukes the node_modules stuff